sherlocked.js

```diff
@@ -0,0 +1,74 @@
+var sherlocked = require('sherlocked');
+
+var PORT = require('./config').PORT;
+
+var environments = [
+    {
+        browserName: 'firefox',
+        platform: 'OS X 10.9'
+    },
+    {
+        browserName: 'chrome',
+        platform: 'OS X 10.9'
+    }
+];
+
+
+var captures = [
+    {
+        name: 'Home Page on Mobile',
+        capture: function(client) {
+            return client.setViewportSize({width: 320, height: 960})
```

Comment:
I'd either drop all the `.func()` calls to their own line or line the subsequent calls up with the first `.`